### ISSUE TITLE
Unity 2018 Issue socket io issue

### ISSUE THREAD

**Issue by mkashifwk** [floatinghotpot/socket.io-unity#37]
Hello, I dragged dlls to my unity assets, it gives me error

/Users/apple/Library/Unity/cache/packages/packages.unity.com/com.unity.package-manager-ui@1.9.11/Editor/Sources/UI/Common/PopupField.cs(61,38): error CS0433: The imported type `System.Func<T,TResult>' is defined multiple times

packages.unity.com/com.unity.textmeshpro@1.2.4/Scripts/Runtime/FastAction.cs(83,27): error CS0433: The imported type `System.Action<T1,T2>' is defined multiple times

**Comment by ATheCoder** [comment]
Yeah, This is the conflict with unity's package Manager UI, Downgrading to Version 2017.1 (When there was no package Manager) will get around this issue.

**Comment by mkashifwk** [comment]
@ATheCoder ok, thanks,

**Comment by LagacheLouis** [comment]
@mkashifwk @ATheCoder  I have discovered a solution, you can remove the package manager ui dependancy in the project in "yourproject"/Packages/manifest.json. It will remove the package manager from your project.

**Comment by bcjordan** [comment]
@LagacheLouis I see, this would make this Socket.io work but then you just can't import new platforms etc from Unity?

**Comment by garzj** [comment]
[This library](https://github.com/dp0ch/Unity-SocketIO-Client) works much better for me.
It saved me from a lot of headaches, so I thought I'll share it with you.
